src/server/task-list/controller.js

```diff
@@ -13,63 +13,29 @@ const buttonText = 'Review and submit'
  */
 export const taskListGetController = {
   handler(req, h) {
-    const origin = Origin.fromState(req.yar.get('origin'))
-    const destination = Destination.fromState()
-    const tests = Tests.fromState()
-    const licence = Licence.fromState(req.yar.get('licence'))
-    const isOriginValid = origin.validate().isValid
-
-    const originGdsTask = buildGdsTaskItem({
-      title: 'Movement origin',
-      initialLink: 'origin/to-or-from-own-premises',
-      summaryLink: 'origin/summary',
-      isValid: isOriginValid,
-      isEnabled: true
-    })
-
-    const destinationGdsTask = buildGdsTaskItem({
-      title: 'Movement destination',
-      initialLink: '#',
-      summaryLink: '#',
-      isValid: destination.validate().isValid,
-      isEnabled: isOriginValid
-    })
+    const application = {
+      origin: Origin.fromState(req.yar.get('origin')),
+      destination: Destination.fromState(),
+      tests: Tests.fromState(),
```

Comment:
I believe we've dropped the conditional values that @joseluisgraa  has implemented.

For instance, the "Movement destination" section (aka `destination`) is enabled if the "Movement Origin" section (aka `origin`) is valid. 

Same dependency on `origin` from `tests`.

Reply:
that is to be added to section page model
isEnabled could just as easily be a function that works out if its enabled or not

at present, we dont have a section model that is able to look at a section outside of itself